_gsocproposals/2022/proposal_Belle2Validation.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Advanced Belle II Software Validation
+layout: gsoc_proposal
+project: Belle II
+year: 2022
+organization: LMU
+---
+
+## Description
+The international Belle II collaboration relies on custom software for the simulation, reconstruction, and analysis of the data recorded by the Belle II detector. The software was developed by hundreds of physicists over more than a decade. The development process will continue for several years to optimize algorithms for better physics performance or resource efficiency, to adjust the code to technological evolutions, or to implement new ideas.
+
+With such a large, diverse, and quickly changing development team the quality assurance of the software is a challenge. Belle II has implemented several mechanisms to monitor the software quality. An essential tool is the high-level validation. It runs algorithms on large datasets and produces plots of distributions of quantities that are relevant for physics analyses. The distributions are displayed on a web page and compared with previous versions and a reference. With this degradations can be quickly identified and fixed.
```

Comment:
degradations -> quality degradations ?(to be clear what degradations you are talking about)